static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -1,11 +1,301 @@
+        margin-left: -(gs-span(3) + $gs-gutter);
+    }
+}
+
+.gallery__caption {
+    color: $neutral-4;
+
+    @include mq(desktop) {
+        padding-right: $gs-gutter / 2;
+    }
+
+    a {
+        color: $neutral-4;
+        border-bottom: 1px solid $neutral-2;
+        transition: border-color .15s ease-out;
+
+        &:hover {
+            border-bottom: 1px solid $neutral-7;
+            text-decoration: none;
+        }
+    }
+
+    // This is the gallery caption title
+    > strong:first-child {
+        @include fs-headline(2);
+        color: $neutral-5;
+        display: block;
+        margin-bottom: $gs-baseline/2;
+
+        // The first br is display none after the title because the
+        // title already has the correct margins provided in "fs-headline(2)"
+        & + br {
+            display: none;
+        }
+    }
+}
+
+.gallery__credit {
+    color: $neutral-3;
+
+    @include mq(desktop) {
+        padding-right: $gs-gutter / 2;
+    }
+}
+
+.gallery__img-container {
+    @include mq(desktop) {
+        padding-bottom: 2.5rem;
+    }
+
+    @include mq($until: desktop) {
+        text-align: center;
+
+        img {
+            display: inline-block;
+        }
+    }
+}
+
+// All images should fit within the viewport height
+.gallery__img-container,
+.gallery__img {
+    width: 100%;
+}
+
+.content__labels--gallery,
+.l-side-margins--gallery .content__meta-container,
+.content__headline--gallery,
+.content__standfirst--gallery {
+    @include mq(desktop) {
+        margin-left: gs-span(3) + $gs-gutter;
+        max-width: gs-span(8);
+    }
+}
+
+.gallery__meta-container {
+    .byline,
+    .content__dateline {
+        border: 0;
+        min-height: 0;
+        padding: 0;
+    }
+}
+
+.l-side-margins--gallery {
+    .content__meta-container {
+        border: 0;
+        position: relative;
+        width: auto;
+        min-height: auto;
+        margin-bottom: $gs-baseline;
+    }
+
+    .meta__comment-count--top {
         display: none;
     }
+
+    .meta__comment-count--bottom {
+        display: block;
+        float: none;
+        padding-top: $gs-baseline/2;
+
+        .commentcount {
+            position: static;
+        }
+
+        .commentcount__label {
+            display: inline;
+        }
+    }
+
+    .witness-cta {
+        border: 0;
+    }
+
+    .meta__extras {
+        border-top: 1px dotted $media-mute;
+        border-bottom: 1px dotted $media-mute;
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .meta__social {
+        border: 0;
+
+        .social--top {
+            padding: 8px 0 0;
+
+            .social__item {
+                padding-right: $gs-gutter/4;
+                min-width: auto;
+            }
+        }
+    }
+
+    .meta__numbers {
+        border: 0;
+        flex-shrink: 0;
+        position: static;
+        min-width: gs-span(1);
+    }
 }
 
+// TODO:
+// When paid advertisement features are redesigned, delete below styling
```

Comment:
@blongden73 all this code should be deleted with the new design for paid advertisements that you are coming up with! 😉